README.md

```diff
@@ -16,7 +16,7 @@ This repository contains the current consensus genome-scale metabolic model of _
 
 |Taxonomy | Template Model | Reactions | Metabolites| Genes |
 |:-------:|:--------------:|:---------:|:----------:|:-----:|
-|_Saccharomyces cerevisiae_|[Yeast 7.6](https://sourceforge.net/projects/yeast/)|3928|2666|1133|
+|_Saccharomyces cerevisiae_|[Yeast 7.6](https://sourceforge.net/projects/yeast/)|3949|2680|1133|
```

Comment:
in your PR summary you mention there are 8 extra metabolites, however this shows that there are 14. Which one is it?

Reply:
8 extra metabolites from metabolomics data, other 6 mets are existed in the model with another compartment.